.github/workflows/test-run-enos-scenario-matrix.yml

```diff
@@ -168,3 +168,12 @@ jobs:
           rm -rf /tmp/enos*
           rm -rf ./enos/support
           rm -rf ./enos/.enos
+      # Send Slack notification to #feed-vault-enos-failures for if the 'run_retry' step fails.
+      # There is an incoming webhook set up on the "Enos Vault Failure Bot" Slackbot https://api.slack.com/apps/A05E31CH1LG/incoming-webhooks
+      - name: Send Slack notification on Enos failure
+        uses: hashicorp/actions-slack-status@v1
+        if: ${{ always() }}
+        with:
+          failure-message: "An Enos scenario run failed."
+          status: ${{steps.run_retry.outcome}}
```

Comment:
I assume this action magically knows to only send if the outcome is failure? I was thinking it would be nice to send if either the run, cleanup, or retry failed. Is that possible?

Reply:
If you only include a `failure-message` and not a `success-message`, it will only run when the `status` that you include is set to `failure`. So in this case it will only send the `failure-message` if the `run_retry` step fails. I only included `run_retry` because I didn't think it was useful for us to get double notifications if both `run` and `run_retry` fail. If `run` fails but `run_retry` succeeds, I assume the reason is probably some flakiness out of our control.

I'm not sure if I can add multiple outcomes to `status`, but I can test that. If not, we would need to add another slack notification step that checks for failure on the `destroy` step.